components/UserSavedCard.vue

```diff
@@ -1,44 +1,85 @@
 <template>
   <div>
-    <div v-if="isDefaultCard()" class="is-primary">Primary</div>
-    <div class="card-list">
+    <div v-if="!isReadonly" class="card-list">
```

Comment:
always use positive condition first for readability purpose ex v-if="isReadonly"